curve25519-dalek/src/traits.rs

```diff
@@ -421,3 +421,9 @@ pub(crate) trait ValidityCheck {
     /// Checks whether the point is on the curve. Not CT.
     fn is_valid(&self) -> bool;
 }
+
+// Reduce trait 
+
+pub trait Reduce<T>{
```

Comment:
Why is this re-defined here? Isn't the point of the referenced issue that we should be implementing the `Reduce` trait from `elliptic_curve`?

Reply:
Also note we'll have another breaking release of `elliptic-curve` out soon, probably early 2025, so it would probably make sense to target these changes along #676